openid-4-verifiable-presentations-1_0.md

```diff
@@ -1088,6 +1164,12 @@ Implementations MUST follow [@!BCP195].
 
 Whenever TLS is used, a TLS server certificate check MUST be performed, per [@!RFC6125].
 
+# Privacy Considerations
+
+## Authorization Requests with Request URI
+
+The Wallet MUST NOT send personally identifiable information (PII) or any other data that could be used for fingerprinting to the Request URI in order to prevent user tracking. 
```

Comment:
I guess "any other data" also includes HTTP headers such as "User-Agent" and "Client"... I'm also wondering whether the specifications should explicitly suggest excluding them for Wallets due to privacy considerations.

Reply:
It's also uncommon to see that libraries works with cookies exactly in the same way like browsers do. So, at some degree, it all applicable to the "Cookies" request header as well.

Just as an example take a quick look at https://docs.oracle.com/en%2Fjava%2Fjavase%2F21%2Fdocs%2Fapi%2F%2F/java.base/java/net/CookieManager.html

Reply:
It applies to all data, independent of the part of the request it is passed in or the representation. Can you please suggest text?

Reply:
@tlodderstedt, let's leave it as is then... it would make more sense to cover these nuances in conformance tests for the spec.